code/modules/mob/living/living.dm

```diff
@@ -988,6 +988,20 @@ Thanks.
 						BD.attack_hand(usr)
 					C.open()
 
+	if(src.loc && istype(src.loc, /obj/item/mecha_parts/mecha_equipment/tool/jail))
+		var/breakout_time = 0.5 //30 seconds by default
```

Comment:
What I would suggest is that you move this to be directly `0.5 MINUTES` or `30 SECONDS` in the do_after, because you aren't making any changes to this value in the code, and it makes the code less readable.